HopsanGUI/Widgets/ModelWidget.cpp

```diff
@@ -121,7 +121,7 @@ ModelWidget::ModelWidget(ModelHandler *pModelHandler, CentralTabWidget *pParentT
     mpGraphicsView  = new GraphicsView(this);
     //mpGraphicsView->setScene(mpToplevelSystem->getContainedScenePtr());
 
-    mpLogDataHandler = new LogDataHandler2(this);
+    mpLogDataHandler = QSharedPointer<LogDataHandler2>(new LogDataHandler2(this));
```

Comment:
To enure that the object is deleted later, supply deleter (I think that we want that). See https://stackoverflow.com/a/12624043

Reply:
Good point, I will fix it.